.gitignore

```diff
@@ -411,6 +411,7 @@ modules.xml
 # Project specific ignores
 /build.properties
 /data/
+**/respondr/defaultSkin.css
```

Comment:
What's the point of this line?  The generated `defaultSkin.css` file will be in the `build/` directory, which is already ignored.